proto/buf.gen.yaml

```diff
@@ -11,11 +11,11 @@ plugins:
     out: generated-go
     opt:
       - paths=source_relative
-  - remote: buf.build/stephenh/plugins/ts-proto
+  - plugin: buf.build/community/stephenh-ts-proto
```

Comment:
Newer versions of the `buf` cli will use `plugin` key for both local and remote plugin references.

The [buf.build/community](https://buf.build/community) organization is maintained by @bufbuild, and the plugins are located here:

- https://buf.build/community/pseudomuto-doc
- https://buf.build/community/stephenh-ts-proto